library/core/class.session.php

```diff
@@ -26,6 +26,9 @@ class Gdn_Session {
     /** @var object A User object containing properties relevant to session */
     public $User;
 
+    /** @var Gdn_SQLDriver Contains the sql driver which is set in method stash. */
+    public $sql;
```

Comment:
As stated by http://docs.vanillaforums.com/developer/contributing/coding-standard/#4-3-properties
No underscore. Never hesitate to ask it is not bothering us at all.